LEAF_Request_Portal/form.php

```diff
@@ -561,8 +562,17 @@ public function getIndicatorLog($indicatorID, $series, $recordID)
                 }
             }
 
+            $is_sensitiveVar = array(
+                ':indicatorID' => $indicatorID,
+            );
+
+            $is_sensitive = $this->db->prepared_query(
+                'SELECT is_sensitive FROM indicators
+                WHERE indicatorID=:indicatorID', $is_sensitiveVar
+            );
```

Comment:
In this instance, we should include the `is_sensitive` column along with the query on line 531. This can be done with another LEFT JOIN on the `indicators` table. Only the `is_sensitive` column should be pulled from the `indicators` table, so you'll probably have to select every column by name and table in that query. Something like:

```sql
SELECT h.recordID, h.indicatorID, /* other data_history columns */, i.is_sensitive 
   FROM data_history h, indicators i 
   LEFT JOIN indicators USING (indicatorID) blahblahblah
```

This way all the relevant data is contained in that initial query and the load on the database is reduced. As it is now, this is looping over all the `data_history` results and spawning another database call for each result, which could potentially be a lot.